settings.py

```diff
@@ -748,11 +748,8 @@ def _color_scheme_completions(view):
             if any(hide in scheme_path for hide in hidden):
                 continue
             _, package, *_, file_name = scheme_path.split("/")
-            item = (
-                "{}  \tPackage: {}".format(file_name, package),
```

Comment:
Themes are specified by the basename alone, because they can be patched by other packages. Color schemes on the other hand are immutable.

I agree that the Package adds some noise and it's probably not necessary, though imo it also adds value when you copied a color scheme to your User package for example. Maybe just removing `Package:` from the hint is enough here?